src/main/java/duke/Deadlines.java

```diff
@@ -0,0 +1,47 @@
+package duke;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task to be added to Duke with a deadline.
+ */
+public class Deadlines extends Task {
+
+    protected String description;
```

Comment:
Should this line be removed? This field does not even seem to be used within this file, and it looks like the description is supposed to be stored in Task.